src/app/components/ReclaimEscrowForm/index.tsx

```diff
@@ -49,8 +49,19 @@ export const ReclaimEscrowForm = memo((props: Props) => {
       transactionActions.reclaimEscrow({
         type: 'reclaimEscrow',
         amount: Number(amount),
-        validator: props.address,
         shares: shares,
+        validator: props.address,
+      }),
+    )
+  }
+
+  const reclaimAll = () => {
+    dispatch(
+      transactionActions.reclaimEscrow({
+        type: 'reclaimEscrow',
+        amount: Number(props.maxAmount) / 10 ** 9,
+        shares: Number(props.shares) / 10 ** 9,
```

Comment:
Further refactored in https://github.com/oasisprotocol/oasis-wallet-web/pull/932 to keep precision

Reply:
FYI shares should be treated as integer too, see #79.

Reply:
too many things are using gigashares lol